packages/react-components/CHANGELOG.md

```diff
@@ -5,11 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.4.0] - 2023-11-09
```

Comment:
Outdated :) I recommend doing the version bumps (both package.json and here) in a separate commit done just before releasing.